src/containers/ProjectDashboardContainer.jsx

```diff
@@ -43,9 +44,11 @@ class ProjectDashboardContainer extends Component {
   }
 
   handleSelect({ option }) {
+    const { actions } = this.props;
     this.setState({
       option,
     });
+    actions.createTranslation('project_contents', option.value);
```

Comment:
At the moment, creating a new translation is called from `selectTranslation`, if the translation isn't found in the `translations` array. https://github.com/zooniverse/pandora/pull/19/files#diff-26d3d8bb803fd9662fecef23eb6074fbR124